specification/translation/Azure.AI.DocumentTranslation/client.tsp

```diff
@@ -41,49 +41,164 @@ interface SingleDocumentTranslationClient {
   documentTranslate is DocumentTranslation.DocumentTranslationOperations.documentTranslate;
 }
 
-// model overrides
-@@access(DocumentTranslation.DocumentTranslateResult,
-  Access.internal,
+
+@@clientName(DocumentTranslation.DocumentTranslateBody.body,
+  "documentTranslateContent",
+  "csharp,java"
+);
+
+@@clientName(DocumentTranslation.StartTranslationBody.body,
+  "startTranslationDetails",
+  "csharp,java"
+);
+
+@@clientName(DocumentTranslation.StartTranslationBody.body,
+  "startTranslationDetails",
+  "csharp,java"
+);
+
+
+@@access(DocumentTranslationClient.getSupportedFormats, Access.internal, "python");
+@@access(DocumentTranslation.FileFormat, Access.public);
+@@usage(DocumentTranslation.FileFormat, Usage.output);
+@@access(DocumentTranslation.FileFormatType, Access.public);
+@@usage(DocumentTranslation.FileFormatType, Usage.output);
```

Comment:
Is this applicable cross language? The way it's defined now implies that it is

Also, is FileFormatType not public by default?

Reply:
I had to make the operation that uses them internal (which I guess makes all associated models internal as well):

@@access(DocumentTranslationClient.getSupportedFormats, Access.internal, "python");

because we split this method into two on the client in Python. These should be specific to Python, though, I'll make that change.